package.json

```diff
@@ -24,9 +21,5 @@
   "bugs": {
     "url": "https://github.com/highlightjs/highlightjs-dylan/issues"
   },
-  "homepage": "https://github.com/highlightjs/highlight-dylan#readme",
-  "devDependencies": {
```

Comment:
Technically you may want to have a look at peer dependencies, but this is likely fine either way.

https://nodejs.org/en/blog/npm/peer-dependencies/

Reply:
OK will do that separately.